tests/unit/src/main/scala/tests/CompileResult.scala

```diff
@@ -8,8 +8,10 @@ case class CompileResult(
     textDocuments: Semanticdb.TextDocuments,
     isSuccess: Boolean
 ) {
-  def textDocument: Semanticdb.TextDocument = {
-    textDocuments.getDocuments(0)
+  def textDocument: Option[Semanticdb.TextDocument] = {
+    Option.when(textDocuments.getDocumentsCount() > 0) {
```

Comment:
This was previously crashing, likely terminating the benchmark prematurely.

```
[info] <failure>
[info] java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
[info]  at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
[info]  at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
[info]  at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
[info]  at java.base/java.util.Objects.checkIndex(Objects.java:372)
[info]  at java.base/java.util.ArrayList.get(ArrayList.java:459)
[info]  at java.base/java.util.Collections$UnmodifiableList.get(Collections.java:1310)
[info]  at com.sourcegraph.semanticdb_javac.Semanticdb$TextDocuments.getDocuments(Semanticdb.java:669)
[info]  at tests.CompileResult.textDocument(CompileResult.scala:12)
[info]  at benchmarks.CompileBench.$anonfun$compileSemanticdb$1(CompileBench.scala:65)
[info]  at benchmarks.CompileBench.$anonfun$compileSemanticdb$1$adapted(CompileBench.scala:64)
[info]  at benchmarks.CompileBench$.$anonfun$foreachSource$2(CompileBench.scala:94
[info]  at benchmarks.CompileBench$.$anonfun$foreachSource$2$adapted(CompileBench.scala:82)
[info]  at scala.meta.internal.io.PlatformFileIO$.withJarFileSystem(PlatformFileIO.scala:80)
[info]  at scala.meta.internal.io.FileIO$.withJarFileSystem(FileIO.scala:44)
[info]  at benchmarks.CompileBench$.$anonfun$foreachSource$1(CompileBench.scala:82
[info]  at benchmarks.CompileBench$.$anonfun$foreachSource$1$adapted(CompileBench.scala:80)
[info]  at scala.collection.immutable.List.foreach(List.scala:333)
[info]  at benchmarks.CompileBench$.foreachSource(CompileBench.scala:80)
```

Reply:
I thought I'd check which dependency causes this:

```
[info] ArraySeq(edu/umd/cs/findbugs/annotations/When.java, edu/umd/cs/findbugs/annotations/UnknownNullness.java, edu/umd/cs/findbugs/annotations/SuppressWarnings.java, edu/umd/cs/findbugs/annotations/SuppressFBWarnings.java, edu/umd/cs/findbugs/annotations/ReturnValuesAreNonnullByDefault.java, edu/umd/cs/findbugs/annotations/Priority.java, edu/umd/cs/findbugs/annotations/PossiblyNull.java, edu/umd/cs/findbugs/annotations/package-info.java, edu/umd/cs/findbugs/annotations/OverrideMustInvoke.java, edu/umd/cs/findbugs/annotations/Nullable.java, edu/umd/cs/findbugs/annotations/NoWarning.java, edu/umd/cs/findbugs/annotations/NonNull.java, edu/umd/cs/findbugs/annotations/ExpectWarning.java, edu/umd/cs/findbugs/annotations/DischargesObligation.java, edu/umd/cs/findbugs/annotations/DesireWarning.java, edu/umd/cs/findbugs/annotations/DesireNoWarning.java, edu/umd/cs/findbugs/annotations/DefaultAnnotationForParameters.java, edu/umd/cs/findbugs/annotations/DefaultAnnotationForMethods.java, edu/umd/cs/findbugs/annotations/DefaultAnnotationForFields.java, edu/umd/cs/findbugs/annotations/DefaultAnnotation.java, edu/umd/cs/findbugs/annotations/CreatesObligation.java, edu/umd/cs/findbugs/annotations/Confidence.java, edu/umd/cs/findbugs/annotations/CleanupObligation.java, edu/umd/cs/findbugs/annotations/CheckReturnValue.java, edu/umd/cs/findbugs/annotations/CheckForNull.java) returned an empty textDocument
```

Reply:
This is quite strange, because when I index it manually it works:

```
sbt> cli/run index-dependency --dependency com.google.code.findbugs:jsr305:3.0.2
```

<img width="946" alt="CleanShot 2023-07-26 at 14 34 52@2x" src="https://github.com/sourcegraph/scip-java/assets/1052965/429eb742-9b4a-4c49-b0d7-f6f73b2ddead">